docs/zkapps/roadmap.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: zkApps Roadmap
+hide_title: true
+sidebar_label: Roadmap
+---
+
+import Subhead from '@site/src/components/common/Subhead';
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# zkApps Roadmap
+
+<Subhead>High-level overview of features available now, next, or later</Subhead>
+
+## Now
+
+### Mina zkApp CLI
+
+- [Scaffold projects & files](/zkapps/how-to-write-a-zkapp#option-b-start-your-own-project)
+- [Example projects](/zkapps/how-to-write-a-zkapp#option-a-start-with-an-example-recommended)
+- [Deployment](/zkapps/how-to-deploy-a-zkapp)
+- Create a project with an accompanying UI — SvelteKit, NextJS, or NuxtJS
+
+:::note
+
+Install via `npm install -g zkapp-cli`. Use `zk --help` to see all commands.
```

Comment:
Suggestion to clarify what this installs: "Install the Mina zkApp CLI via 'npm install...'"

Reply:
Thanks. I rephrased to emphasize why I'm mentioning this:

> To view all commands offered by the zkApp CLI, install via `npm install -g zkapp-cli` and use `zk --help`.